utils/ioprovenance.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+Utility funcitons for provenance of feature files.
+
+Authors: Phillip J. Wolfram
+Last Modified: 09/29/2016
+"""
+
+import os
+import sys
+import socket
+import subprocess
+import numpy as np
+import datetime
+
+def provenance_command(): #{{{
+    cwd = os.getcwd()
```

Comment:
Did you think we should remove this provenance?  Also, do we need a history scrubber script too in this PR to help alleviate privacy concerns?